examples/single_case_lesbrary_ri_based_calibration.jl

```diff
@@ -0,0 +1,186 @@
+# # CAKTE calibration with Ensemble Kalman Inversion using LESbrary data 
+
+# ## Install dependencies
+
+# ```julia
+# using Pkg
+# pkg"add ParameterEstimocean, Oceananigans, CairoMakie"
+# ```
+
+using Oceananigans
+using Oceananigans.Units
+using ParameterEstimocean
+using LinearAlgebra, CairoMakie, DataDeps, Distributions
+
+using Oceananigans.TurbulenceClosures: RiBasedVerticalDiffusivity
+
+# # Using LESbrary data
+#
+# `ParameterEstimocean.jl` provides paths to synthetic observations
+# derived from high-fidelity large eddy simulations. In this example, we illustrate
+# calibration of a turbulence parameterization to one of these simulations:
+
+datapath = datadep"two_day_suite_1m/strong_wind_instantaneous_statistics.jld2"
+times = [2hours, 12hours, 24hours]
+field_names = (:b, :u, :v)
+transformation = ZScore()
+regrid = (1, 1, 32)
+
+observations = SyntheticObservations(datapath; field_names, times, transformation, regrid)
+
+# Let's take a look at the observations. We define a few
+# plotting utilities along the way to use later in the example:
+
+colorcycle = [:black, :red, :darkblue, :orange, :pink1, :seagreen, :magenta2]
+markercycle = [:rect, :utriangle, :star5, :circle, :cross, :+, :pentagon]
+
+function make_figure_axes()
+    fig = Figure(resolution=(1200, 400))
+    ax_b = Axis(fig[1, 1], xlabel = "Buoyancy \n[cm s⁻²]", ylabel = "z [m]")
+    ax_u = Axis(fig[1, 2], xlabel = "x-velocity, u \n[cm s⁻¹]")
+    ax_v = Axis(fig[1, 3], xlabel = "y-velocity, v \n[cm s⁻¹]")
+    return fig, (ax_b, ax_u, ax_v)
+end
+
+function plot_fields!(axs, b, u, v, label, color)
+    z = znodes(Center, b.grid)
+    ## Note unit conversions below, eg m s⁻² -> cm s⁻²:
+    lines!(axs[1], 1e2 * interior(b, 1, 1, :), z; color, label)
+    lines!(axs[2], 1e2 * interior(u, 1, 1, :), z; color, label)
+    lines!(axs[3], 1e2 * interior(v, 1, 1, :), z; color, label)
+    return nothing
+end
+
+# And then plot the evolution of the observed fields,
+
+fig, axs = make_figure_axes()
+
+for (i, t) in enumerate(times)
+    fields = map(name -> observations.field_time_serieses[name][i], field_names)
+    plot_fields!(axs, fields..., "t = " * prettytime(t), colorcycle[i])
+end
+
+[axislegend(ax, position=:rb, merge=true, fontsize=10) for ax in axs]
+
+save("lesbrary_synthetic_observations.svg", fig); nothing # hide
+
+# ![](lesbrary_synthetic_observations.svg)
+
+# Behold, boundary layer turbulence!
+# 
+# # Calibration
+#
+# Next, we build a simulation of an ensemble of column models to calibrate
+# CATKE using Ensemble Kalman Inversion. We configure CATKE without convective
+# adjustment and with constant (rather than Richardson-number-dependent)
+# diffusivity parameters.
```

Comment:
```suggestion
# Next, we build a simulation of an ensemble of column models to calibrate
# the closure using Ensemble Kalman Inversion. We configure the closure without convective
# adjustment and with constant (rather than Richardson-number-dependent)
# diffusivity parameters.
```